packages/mendel-generator-prune/README.md

```diff
@@ -0,0 +1,24 @@
+# mendel-generator-prune
+
+This generator removes dangling files, such as files required initially and later not used because of some transformation (examples would be `requires` removed after `envify` and some CSS-in-JS dependency. It also assigns `NORMALIZED_ID` to files exposed in other bundles.
```

Comment:
Suggestion here: `NORMALIZED_ID` is the relative path to either `base` or any variation.